docs/config.rst

```diff
@@ -739,6 +739,14 @@ Features
 
    .. versionadded:: 2.2
 
+.. attribute:: LP_ENABLE_PATH
+   :type: bool
+   :value: 1
+
+   Display the current working directory.
+
+   .. versionadded:: 2.2
```

Comment:
Maybe add a note on this that despite it being a new option, the feature has existed since the beginning?
```suggestion
   .. versionadded:: 2.2
      Before this version, this feature was always enabled.
```